UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/UI_Ai.cs

```diff
@@ -69,6 +69,8 @@ public class UI_Ai : MonoBehaviour
 
 		private bool focusCheck;
 
+		private CooldownInstance stateCooldown = new CooldownInstance { defaultTime = 5f };
```

Comment:
Might be nice to overload the constructor w/ `defaultTime` parameter?

Reply:
Done